crawl-ref/source/dat/des/builder/shops.des

```diff
@@ -1301,6 +1301,36 @@ xx'''NbWbb..xx
    xxxxxxxx
 ENDMAP
 
+# A reference to the special Oracle level from NetHack.
+# Like in NetHack, players can purchase readings from the Oracle. In a sense.
+NAME:   nicolae_nh_oracle_at_delphi
+TAGS:   transparent no_rotate
+DEPTH:  D:5-9
+WEIGHT: 1
```

Comment:
Why the Depth restriction? I know it's part of the reference but with this tight range and weight 1 this vault is going to be impossible to place for the level builder... IMO D, Depths, Elf is a good range for something like this.

Reply:
> Why the Depth restriction?

Solely for the reference, but you're right. Even the players who get the reference aren't going to know this vault's depth range. I'll fix it up.